src/renderer/src/components/PasswordStrenght.tsx

```diff
@@ -0,0 +1,57 @@
+import { useTranslation } from 'react-i18next'
+import { regexGood, regexStrong } from '@renderer/constants/passwordStrenght'
+import { StyleHelper } from '@renderer/helpers/StyleHelper'
+
+type TProps = {
+  password: string
+}
+
+export const PasswordStrenght = ({ password }: TProps) => {
+  const { t } = useTranslation('components', { keyPrefix: 'passwordStrenght' })
+
+  const isPasswordGood = () => regexGood.test(password)
+  const isPasswordStrong = () => regexStrong.test(password)
+  const isPasswordTooShort = () => password.length <= 4
```

Comment:
It does not need to be a function, so you don't need to call every time

```suggestion
  const isPasswordGood = regexGood.test(password)
  const isPasswordStrong = regexStrong.test(password)
  const isPasswordTooShort = password.length <= 4
```